airbyte-ci/connectors/pipelines/CONTRIBUTING.md

```diff
@@ -0,0 +1,336 @@
+| [`format`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py#L32)          | Check that the full repo is correctly formatted          | `airbyte-ci format check all`                 | [`airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py#L78)                                       |
+| [`format`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py#L32)          | Format the whole repo                                    | `airbyte-ci format fix all`                   | [`airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/commands.py#L101)                                      |
+| [`test`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/commands.py#L107)             | Run tests on internal poetry packages                    | `airbyte-ci test`                             | [`airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/commands.py`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/commands.py#L107)                                          |
+| [`poetry`](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/poetry/commands.py#L33)          | Publish a poetry package to PyPi                         | `airbyte-ci poetry publish`                   | [`airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/poetry/publish/commands.py`](https:github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/poetry/publish/commands.py#L69)                         |
+
+## How to install the package for development
+
+There are multiple way to have dev install of the tool. Feel free to grab the one you prefer / which works for you.
+**Please note that all the install mode lead to an editable install. There's no need to re-install the tool following a code change**.
+
+### System requirements
+* `Python` > 3.10
+* [`Poetry`](https://python-poetry.org/) or [`pipx`](https://github.com/pypa/pipx)  
+
+### Installation options
+
+#### With `make`
+```bash
+ # From airbyte repo root:
+ make tools.airbyte-ci-dev.install
+ ```
+
+#### With `pipx`
+```bash
+# From airbyte-ci/connectors/pipelines:
+pipx install --editable --force .
+```
+
+#### With `poetry`
+```bash
```

Comment:
```suggestion
⚠️ This places you in a python environment specific to airbyte-ci. This can be a problem if you are developing airbyte-ci and testing/using your changes in another python project.

```bash
```